packages/cli/src/constructs/tcp-monitor.ts

```diff
@@ -52,10 +69,29 @@ export interface TcpMonitorProps extends MonitorProps {
   request: TcpRequest
   /**
    * The response time in milliseconds where a check should be considered degraded.
+   * TCP checks have lower thresholds than HTTP checks due to protocol differences.
+   * 
+   * @defaultValue 4000
+   * @minimum 0
+   * @maximum 5000
```

Comment:
These are correct for the public API but it looks like the CLI endpoint does not enforce the same values. These are the effective values currently:

```
   * @defaultValue 10000
   * @minimum 0
   * @maximum 300000
```

This should probably be fixed in the backend.